setup.cfg

```diff
@@ -1,7 +1,10 @@
 [easy_install]
 
-[pytest]
+[tool:pytest]
```

Comment:
Search for `tool:pytest` in the [pytest changelog](https://docs.pytest.org/en/latest/changelog.html#id76) for more info.